string header
uint16 protocol_version
uint8 packet_number
uint8 players_per_team
string game_type
string state
uint8 first_half
uint8 kick_off_team
string secondary_state

uint8 teamPerformingSubMode
string submode

uint8 drop_in_team
uint16 drop_in_time
uint16 secs_remaining
uint16 secondary_time

#atributos del equipo en general
uint8 teamNumber
uint8 teamColor
uint8 score 
uint8 penaltyShotCounter
uint8 coachSequence




#atributos de cada robot 
string penalty
uint8 penaltyTime
uint8 warnings
uint8 yellowCards
uint8 redCards
uint8 gkFlag

#atributos importantes
string robotPlayState
uint8 robotPlayStateInt